src/utils/hooks/useLoadTemplate.tsx

```diff
@@ -46,14 +46,13 @@ const useLoadTemplate = (props: useLoadTemplateProps) => {
       return
     }
 
-    const { id, code, name, startMessage, submissionMessage, templateSections } = template
+    const { id, code, name, startMessage, templateSections } = template
```

Comment:
Just wondering what the reason for returning "submissionMessage" from AppStages is, rather than here?

Reply:
I just thought that it would be better as a single-hook call in the Submission page - that gets all that is related to it. And calling this hook would be only needed when you actually want to see all the elements listed for a template.

Reply:
Right, and the submission message isn't used anywhere else, is it. Cool.